packages/react-next/src/components/OverflowSet/OverflowButton.tsx

```diff
@@ -0,0 +1,5 @@
+import { getStyles } from './OverflowSet.styles';
+import { styled } from '../../Utilities';
+import { OverflowButtonBase } from './OverflowButton.base';
+
+export const OverflowButton = styled(OverflowButtonBase, getStyles, undefined, { scope: 'OverflowButton' });
```

Comment:
The styles for OverflowButton is off because the we are missing passing the className in Base (by calling `getClassNames`)

here is my suggestion:
Remove this file (styled `OverflowButton`, and just use `OverflowButtonBase` from `OverflowSet`) but make sure pass it the className)